docs/kanvas-packages/workflows/rules.md

```diff
@@ -0,0 +1,124 @@
+# The Rule
+
+The rules are the main feature in this. The rule is compose for the Rule, the actions trigered and the conditions. Each Actions have the specifics System modules,
```

Comment:
"A rule is composed of the rule itself, the actions to trigger and the conditions for triggering the action. Each action links to a specific System Module."